beets/dbcore/query.py

```diff
@@ -502,7 +505,7 @@ def clause_with_joiner(
         return clause, subvals
 
     def __repr__(self) -> str:
-        return f"{self.__class__.__name__}({self.subqueries!r})"
+        return f"{self.__class__.__name__}({repr(self.subqueries)})"
```

Comment:
cc @Serene-Arc
<https://github.com/beetbox/beets/pull/5337#discussion_r1658186214>

Reply:
@snejus My reading of the PEP for f-strings led me to believe the opposite? [They claim](https://peps.python.org/pep-0498/#s-r-and-a-are-redundant) that they support the old calls, `!r` etc, to maintain compatibility.

Reply:
Good shout @Serene-Arc. What I gathered from that section is that they aren't __required__ anymore since they can be replaced by the equivalent expressions, which was not supported previously.

Regarding the preference, from what I've seen in the wild Python community tends to use the native syntax supported by the f-string instead of an equivalent expression, if possible, for example:

* Datetime
```python
dt = datetime.now()
f"Datetime: {dt:%F %T}"
# vs
f'Datetime: {dt.strftime("%F %T")}'
```

* Indentation
```python
txt = "text"
f"{txt:>10}"
# vs
f'{" " * (10 - len(txt))}{txt}'
```

In a similar way, I prefer `!r` over `repr` call since there's no need to use an expression and it's more concise.

I asked perplexity to see what it thinks: https://www.perplexity.ai/search/should-r-or-repr-call-be-prefe-YGZ43OrrTGOYR2eNjl7QeQ